doc/man/restic-forget.1

```diff
@@ -72,7 +72,7 @@ Exit status is 0 if the command was successful, and non\-zero if there was any e
 
 .PP
 \fB\-c\fP, \fB\-\-compact\fP[=false]
-	use compact format
+	compact printed output
```

Comment:
Please remove this change, as the manpages are generated automatically by the release build scripts.